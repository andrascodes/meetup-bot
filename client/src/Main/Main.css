.Main {
  width: 100%;
  position: relative;
  padding: 50px 0 0 160px;
  text-align: left;
}