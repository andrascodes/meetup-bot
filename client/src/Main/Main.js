import React from 'react'

export const Main = props => (
  <div className="Main">
    <div className="container-fluid">
      {props.children}
    </div>
  </div>
)
