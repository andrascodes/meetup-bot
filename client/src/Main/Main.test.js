import React from 'react'
import renderer from 'react-test-renderer'

import { Main } from './Main'

describe('Main', () => {

  test('renders correctly', () => {
    let tree = renderer.create(
      <Main></Main>
    ).toJSON()
    expect(tree).toMatchSnapshot()
    
    tree = renderer.create(
      <Main>
        <h1>Children</h1>
      </Main>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })

})