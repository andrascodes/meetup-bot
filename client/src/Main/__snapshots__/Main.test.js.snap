exports[`Main renders correctly 1`] = `
<div
  className="Main" />
`;

exports[`Main renders correctly 2`] = `
<div
  className="Main">
  <h1>
    Children
  </h1>
</div>
`;
